gnd/src/main/java/com/google/android/gnd/MainViewModel.java

```diff
@@ -42,7 +44,12 @@
 @SharedViewModel
 public class MainViewModel extends AbstractViewModel {
 
+  /** The window insets determined by the activity. */
+  @Hot(replays = true)
   private final MutableLiveData<WindowInsetsCompat> windowInsets = new MutableLiveData<>();
+
+  /** The state of sign in progress dialog visibility. */
+  @Hot(replays = true)
```

Comment:
Same as above.
Annotation already added to the public accessible method. Let's keep it at only one place.

Reply:
I think we should annotation both fields and accessors. In many case the class' impl will interact with the field directly (not via the public accessor), in which case it will be important to understand the observable's semantics.